required/graphics/graphics.dtx

```diff
@@ -1050,7 +1050,8 @@
 %
 %    \begin{macrocode}
 \edef\Gin@gzext{\detokenize{gz}}
-%    \edef{macrocode}
+%    \end{macrocode}
+% \end{macro}
```

Comment:
strange that that hasn't blown up during production of the documentation.

Reply:
Since `\edef{macrocode}` did not terminate the `macrocode` environment, the following lines were printed verbatim in the documentation. These lines contain one `\end{macro}` and two `\begin{macro}`, so the missing `\end{macro}` did not cause an error.

Reply:
thought it must have been something along those lines, but that wasn't visible in the diff fragment ... was just musing aloud really